### ISSUE TITLE
Could not resolve peer dependency tslib@1.14.1

### ISSUE THREAD

**Issue by sw1tchdev** [bikecoders/ngx-deploy-npm#535]
Hello, when installing ngx-deploy-npm 7.0.0 the following error occurs:

```
npm ERR! Found: tslib@2.6.1
npm ERR! node_modules/tslib
npm ERR!   dev tslib@"^2.3.0" from the root project
npm ERR!   tslib@"^2.3.0" from @nx/devkit@16.6.0
npm ERR!   node_modules/@nx/devkit
npm ERR!     peer @nx/devkit@"^16.0.0" from ngx-deploy-npm@7.0.0
npm ERR!     node_modules/ngx-deploy-npm
npm ERR!       dev ngx-deploy-npm@"7.0.0" from the root project
npm ERR!   3 more (nx, @swc-node/register, @swc/helpers)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer tslib@"1.14.1" from ngx-deploy-npm@7.0.0
npm ERR! node_modules/ngx-deploy-npm
npm ERR!   dev ngx-deploy-npm@"7.0.0" from the root project
```

Ngx-deploy-npm v7.0.0 in npm repository has a wrong version `tslib` in peerDependencies [npm view code](https://www.npmjs.com/package/ngx-deploy-npm/v/7.0.0?activeTab=code)

![npm view code](https://github.com/bikecoders/ngx-deploy-npm/assets/70477902/3f855a64-471a-4fb7-9281-0ac2d51d0722)

**Comment by dianjuar** [comment]
That peer dependency is added on compilation time by Nx. It's not on the [current list of peer-dependencies](https://github.com/bikecoders/ngx-deploy-npm/blob/main/packages/ngx-deploy-npm/package.json#L13-L15), and I don't recall configuring anything to add it.

I would investigate what's going on and why that's being added as a peer dependency without our content on the bundle

**Comment by dianjuar** [comment]
It's strange because in the past (V6) it seems to add the right peer dependency
![image](https://github.com/bikecoders/ngx-deploy-npm/assets/7026066/46f97957-b077-4a30-ae63-fd5d59e138c3)

**Comment by sw1tchdev** [comment]
Maybe `@nx/js:tsc` executor trying to get `tslib` from [related dependency](https://github.com/bikecoders/ngx-deploy-npm/blob/main/package-lock.json#L12683)

**Comment by dianjuar** [comment]
This might be the cause of the problem.

https://nx.dev/packages/angular/executors/package#buildableprojectdepsinpackagejsontype

`updateBuildableProjectDepsInPackageJson` option is deprecated. What that build option do was 

> When `updateBuildableProjectDepsInPackageJson` is true, this adds dependencies to either `peerDependencies` or dependencies.

It might be causing to set the wrong `peerDependency`

**Comment by dianjuar** [comment]
Fixed on [V7.0.1](https://www.npmjs.com/package/ngx-deploy-npm/v/7.0.1)
